src/uu/tr/src/tr.rs

```diff
@@ -57,15 +57,29 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
     if !(delete_flag || squeeze_flag) && sets_len < 2 {
         return Err(UUsageError::new(
             1,
-            format!("missing operand after {}", sets[0].quote()),
+            format!(
+                "missing operand after {}\nTwo strings must be given when translating.",
+                sets[0].quote()
+            ),
         ));
     }
 
-    if delete_flag && !squeeze_flag && sets_len > 1 {
-        return Err(UUsageError::new(
-            1,
-            format!("extra operand {}\nOnly one string may be given when deleting without squeezing repeats.", sets[1].quote()),
-        ));
+    let mut msg: String = "extra operand".into();
+    if sets_len > 1 {
+        if delete_flag && !squeeze_flag {
+            msg = format!("{} {}", msg, sets[1].quote());
+            if sets_len == 2 {
+                msg = format!(
+                    "{}\n{}",
+                    msg, "Only one string may be given when deleting without squeezing repeats."
+                );
+            }
+            return Err(UUsageError::new(1, msg));
+        }
+        if sets_len > 2 {
+            msg = format!("{} {}", msg, sets[2].quote());
+            return Err(UUsageError::new(1, msg));
+        }
```

Comment:
I think you can simplify it even more by avoiding `mut` and returning a string from `if sets_len == 2`, something like:
```
let op = sets[1].quote();
let msg = if sets_len == 2 {
    format!(
        "{msg} {op}\n{}",
        "Only one string may be given when deleting without squeezing repeats."
    )
} else {
    format!("{msg} {op}")
};
```

The other changes look good.